### DISCUSSION TITLE
need help for Sabaki baduk on linux

### DISCUSSION THREAD

**Discussion by Thegoman1972** [community/community#70767] [Programming Help]
hello,
Sabaki run witout problem, but accidently i hide the navigation and in can't save load or do anything else since this.
Sommeone knows how can i restore the navigation 

thanks for help

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Thegoman1972 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
